outputs.nix

```diff
@@ -0,0 +1,124 @@
+{
```

Comment:
these days now that flake.nix file is fairly small in lace-ops, we probably should merge outputs.nix with flake.nix file together